src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -2598,7 +2598,6 @@ return [
             'StreamShelves' => [
                 'grpcStreaming' => [
                     'grpcStreamingType' => 'ServerStreaming',
-                    'resourcesGetMethod' => 'getShelves',
```

Comment:
@michaelbausor Is this a breaking change?

Reply:
Yes. When this setting is present, we use the specified method (in this case `getShelves`) to give the user a stream of `Shelf` objects. When it is not present, we just give them a stream of `StreamShelvesResponse` objects.

Reply:
ok, thanks!